src/classes/emitter.ts

```diff
@@ -32,7 +32,9 @@ export function once(emitter: Emitter, eventName: EventName): Promise<any[]> {
 }
 
 export default class Emitter<Events extends EventMap = EventMap> {
-	#events = new Map<EventName, Listener[]>();
+	#events: {
```

Comment:
We can only type objects.

As our keys are only symbols and strings this is fine